### PR TITLE
Login and register

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/Login.js [hunk 1]
```diff
@@ -17,9 +17,19 @@ class Login extends Component {
     this.Login = this.Login.bind(this);
     this.handleChange = this.handleChange.bind(this);
     this.Validate = this.Validate.bind(this);
-
+    this.togglePassword = this.togglePassword.bind(this);
+    this.ToggleNavbar = this.ToggleNavbar.bind(this);
+  }
+  ToggleNavbar() {
+    var Navbar = document.querySelector(".navbar-custom");
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/Login.js]: Use let instead of var

File: skietbaan_fe/src/components/Login.js [hunk 2]
```diff
@@ -95,65 +105,73 @@ class Login extends Component {
     }
   }
 
+  togglePassword() {
+    var x = document.getElementById("PasswordValue");
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/Login.js]: Use descriptive variables

File: skietbaan_fe/src/components/Login.js [hunk 3]
```diff
@@ -95,65 +105,73 @@ class Login extends Component {
...
-                <Input
+                <label className="front-white input-label" for="examplePassword">
+                Password <div className={this.state.invalidPassword ? "invalid-icon":"hidden"}></div></label>
+                <div className="input-container">
+                <input
                   type="password"
                   name="passwordValue"
-                  id="examplePassword"
-                  placeholder="********"
+                  id="PasswordValue"
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/Login.js]: Keep consistency with your naming.
PasswordValue is not in camelCase notation

File: skietbaan_fe/src/components/Login.js (hunk 2 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: fixed

File: skietbaan_fe/src/components/Login.js (hunk 1 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: fixed

File: skietbaan_fe/src/components/Login.js (hunk 3 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: Fixed

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 4]
```diff
@@ -9,8 +9,14 @@
   }
 }
 
-body{
-  background-image: url("../components/assets/Background.png");
+.Page-content{
+  width: auto;
+  height: stretch;
+  background-color: #504f51;
+  width:100%;
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/RegisterStyles.css]: Two width for the same class. Choose one

Comment (ilzeLourens) [reply skietbaan_fe/src/components/RegisterStyles.css]: didnt see that sorry, fixed it

File: skietbaan_fe/src/components/Login.js [hunk 5]
```diff
@@ -17,9 +17,19 @@ class Login extends Component {
     this.Login = this.Login.bind(this);
     this.handleChange = this.handleChange.bind(this);
     this.Validate = this.Validate.bind(this);
-
+    this.togglePassword = this.togglePassword.bind(this);
+    this.ToggleNavbar = this.ToggleNavbar.bind(this);
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/Login.js]: Keep consistency with your naming.
this.ToggleNavbar should be this.toggleNavBar

File: skietbaan_fe/src/components/Login.js [hunk 6]
```diff
@@ -17,9 +17,19 @@ class Login extends Component {
     this.Login = this.Login.bind(this);
     this.handleChange = this.handleChange.bind(this);
     this.Validate = this.Validate.bind(this);
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/Login.js]: Keep consistency with your naming.

File: skietbaan_fe/src/components/Login.js (hunk 5 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: fixed

File: skietbaan_fe/src/components/Login.js (hunk 6 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: fixed

**Pull request merged**